Modules/CaPTkTrainingModule/src/CaPTkTrainingModule.cpp

```diff
@@ -145,6 +145,20 @@ CaPTkTrainingModule::RunThread(
 		folds.toInt() :
 		samples.toInt();
 
+	if (foldsOrSamples < 2 && configuration == 1)
```

Comment:
Can we use enums for the magic numbers 1 and 2?

Reply:
The whole interface for TrainingModule needs some restructuring. I open an issue about it, issue #22. 

I will fix it when I get time